src/content/administration/okteto-license.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Okteto License
+description: This page provides information about how Okteto's license works for the products we provide.
+sidebar_label: Okteto License
+id: okteto-license
+---
+
+Okteto requires a license key to use any of our paid products. We offer two paid products: [SaaS and  Self-Hosted](/saas-vs-self-hosted/). If you use Okteto SaaS then no action is required of you to obtain a license as this is fully managed by Okteto. If you use Okteto Self-Hosted, then you will need to obtain a license and include that in your `config.yaml` when installing Okteto.
```

Comment:
The license can also be provided using Cloud Secrets: https://www.okteto.com/docs/self-hosted/administration/configuration/#cloud-provider-secret. Should we add a mention to that here?

Reply:
Let me open a different PR for that. This PR fell off my radar and I'd like to get it merged. I'll open a new PR to include that language to avoid re-approving 😄